rasa/core/utils.py

```diff
@@ -538,3 +539,13 @@ def _log_and_get_default_number_of_workers():
         f"no `RedisLockStore` endpoint configuration has been found."
     )
     return _log_and_get_default_number_of_workers()
+
+
+def create_sequence(table_name: Text) -> Sequence:
```

Comment:
I'd opt to leave it in the TrackerStore since it's not used by any other classes / modules. Could e.g. by a private `staticmethod`